src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertEditForm.tsx

```diff
@@ -137,7 +135,6 @@ const SavedSearchAlertEditForm: React.FC<SavedSearchAlertEditFormProps> = ({
 
         const transitionToFilters = () => {
           goToFilters()
-          jumpTo("SavedSearchAlertEditForm")
```

Comment:
The `jumpTo` isn't needed anymore because both containers scroll independently. I also reduced the height of the right panel's loading skeleton. This change causes the loading screen to force the container to scroll to the top.